package.json

```
@@ -18,7 +18,7 @@
     "lint:other": "prettier --check '**/*.{md,yaml,yml}'",
     "lint:quiet": "npm run lint:ts -- --quiet && npm run lint:other",
     "lint:ts": "eslint --config .eslintrc.js --ext .ts,.js .",
-    "mocha": "nyc mocha src/test/**/*.{ts,js}",
+    "mocha": "nyc mocha src/test/*.{ts,js} src/test/**/*.{ts,js}",
```

```suggestion
    "mocha": "nyc 'mocha src/test/**/*.{ts,js}'",
```

I believe this will work (but please verify before you merge the PR). The underlying reason is that some OS X or Windows systems ship with an older version of bash that does not do `**` properly. Quoting it disables the expansion in your `$SHELL`, and instead let the command itself (in this case `mocha`) to do the expansion using Node.js, which is consistent.

If my suggested edit is not working, try `'src/test/*.{ts,js}' 'src/test/**/*.{ts,js}'`. But either way, please do quote all expansions in all scripts.

---

Confirmed that works, thank you!